juntagrico/templates/job.html

```diff
@@ -130,11 +122,21 @@ <h3>
                         {% trans "Allen eine Email senden" %}
                     </button>
                 </form>
-                <br/>
             </div>
         </div>
     {% endif %}
-    <div class="row">
+    <div class="row pb-4">
+        <div class="col-md-3">
+            {% trans "Kontakt" %}:
+        </div>
+        <div class="col-md-5">
+                {{ job.type.activityarea.coordinator|richtext|safe }}
+                <a class="ml-1" href="{% url 'contact-member' member_id=job.type.activityarea.coordinator.pk %}">
```

Comment:
this is  a mailto link in order to be consistent with the rest of the system

Reply:
In the related issue you write that the envelope should be used if I'm not mistaken.

Reply:
yes i understand the same:

"so for the moment i would say display the area main coordinator and also the envelope icon to send an email use the existing simple contact form (and sendign mechanism which is used to send a message to the info email address"